recipes/scikit-multiflow/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "scikit-multiflow" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "69fa62479d6b4cf3aa04800fde3c1cf012c3350fff1cc6f5648a0e646ecbfc5d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script:
+    - "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
```

Comment:
I suspect `sortedcontainers`, `pandas`, `matplotlib`, `scipy`, `scikit-learn`, `pytest` and maybe some others aren't needed here. The `host` section should only contain the minimum set of packages needed to run the `setup.py` script.

Reply:
These are set as requirements in `setup.py`. I had to add them to avoid a RunTimeError due to setuptools downloads, however it might work by using pip.

Reply:
Updated in f31ce0c380863ab695530ba37e0b80ed3aa3ec5d

Reply:
Added pytest and pytest-runner in f67502d to avoid RunTimeError due to setuptools

Reply:
Sorry about that, I got a notification in my phone and thought it was a comment so I duplicated your change.

Reply:
That's okay 😄

It seems to be happy with just `pytest-runner` so LGTM 👍